src/shop-account.html

```diff
@@ -0,0 +1,450 @@
+      }
+
+      .google-signin {
+        @apply(--layout-horizontal);
+        @apply(--layout-center-justified);
+        margin-top: 18px;
+        cursor: pointer;
+      }
+
+      .google-signin img {
+        width: 191px;
+        height: 46px;
+      }
+
+      @media (max-width: 767px) {
+        .login-form {
+          width: 100%;
+        }
+      }
+    </style>
+
+    <div class="main-frame">
+      <iron-meta id="account" type="account"></iron-meta>
+
+      <div class="welcome">
+        <span>Thanks for shopping with us! Why don't you create an account and get 10% discount now?</span>
+      </div>
+
+      <div class="subsection thanks" visible$="[[_computeSigningUp(route)]]">
+        <span>Your order has been successfully placed.</span>
+      </div>
+
+      <div class="subsection welcome" visible$="[[_hasAccount]]">
+        <span>Welcome to Polymer Shop, {{profile.email::change}}!</span>
+
+        <div class="logout-section">
+          <shop-button responsive id="logoutButton">
+            <input type="submit" on-click="_logout" value="Logout">
+          </shop-button>
+        </div>
+      </div>
+
+      <div class="login-section" visible$="[[!_hasAccount]]">
+        <form action="#" id="loginForm"
+            is="iron-form"
+            method="post"
+            on-iron-form-presubmit="_willSendRequest">
+
+          <div class="row input-row">
+            <shop-input>
+              <input type="email" id="accountEmail" name="email"
+                  placeholder="Email" required
+                  aria-labelledby="accountEmailLabel"
+                  autocomplete="username" value$="[[prefilledEmail]]"
+                  autofocus$="[[!_computeFocusOnPW(route, prefilledEmail)]]">
+              <shop-md-decorator error-message="Invalid Email" aria-hidden="true">
+                <label id="accountEmailLabel">Email</label>
+                <shop-underline></shop-underline>
+              </shop-md-decorator>
+            </shop-input>
+          </div>
+
+          <div class="row input-row">
+            <shop-input>
+              <input type="password" id="accountPassword" name="password"
+                  placeholder="Password" required
+                  aria-labelledby="accountPasswordLabel"
+                  autocomplete="current-password"
+                  autofocus$="[[_computeFocusOnPW(route, prefilledEmail)]]">
+              <shop-md-decorator error-message="Invalid Password" aria-hidden="true">
+                <label id="accountPasswordLabel">Password</label>
+                <shop-underline></shop-underline>
+              </shop-md-decorator>
+            </shop-input>
+          </div>
+
+          <shop-button responsive id="loginButton">
+            <input type="button" on-click="_onSignUp" value="Sign Up">
+            <input type="button" on-click="_onSignIn" value="Log In">
+          </shop-button>
+
+          <div class="google-signin" on-click="_onGSignIn">
```

Comment:
I've been looking into `google-signin` component but that requires number of changes in order to use it in this code. For example:
- A mechanism to send `id_token` to the server to verify the identity on server side
- A mechanism to provide user profile
- `google-signin` and `google-signin-aware` are tightly coupled. I would separate `google-signin-aware` as a separate component so it can be used without UI.

I can contribute to `google-signin` in the future, but it will be a significant work I would just leave it and keep current implementation.